### ISSUE TITLE
[Bug/Assistance] webshop failed

### ISSUE THREAD

**Issue by nlpcat** [THUDM/AgentBench#56]
when we add the webshop-dev to the start_task.yaml
```
definition:
  import: tasks/task_assembly.yaml

start:
  dbbench-std: 5
  os-std: 5
  webshop-std: 5
```

```
Traceback (most recent call last):
  File "/root/miniconda3/envs/webshop/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/root/miniconda3/envs/webshop/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/root/workspace/src/server/task_worker.py", line 256, in <module>
    asyncio_task = InstanceFactory.parse_obj(conf[args.name]).create()
  File "/root/workspace/src/typings/general.py", line 34, in create
    mod = __import__(path, fromlist=[self.module.split(".")[-1]])
ModuleNotFoundError: No module named 'src.server.tasks.webshop_docker'
```

this seems to be caused by this
`ln: failed to create symbolic link '/root/workspace/src/server/tasks/webshop_docker': File exists
`
I checked the docker image it already has `/root/webshop` so the `ln -s` command won't work

**Comment by nlpcat** [comment]
`kg-std` is actually not in start_task.yaml. so the readme might be kind of confusing so as the issue in the assignments/default.yaml.

**Comment by nlpcat** [comment]
@Longin-Yu @zhc7

**Comment by zhc7** [comment]
> `kg-std` is actually not in start_task.yaml. so the readme might be kind of confusing so as the issue in the assignments/default.yaml.

Thanks for pointing out. We've udpated README.

**Comment by zhc7** [comment]
> when we add the webshop-dev to the start_task.yaml
> 
> ```
> definition:
>   import: tasks/task_assembly.yaml
> 
> start:
>   dbbench-std: 5
>   os-std: 5
>   webshop-std: 5
> ```
> 
> ```
> Traceback (most recent call last):
>   File "/root/miniconda3/envs/webshop/lib/python3.8/runpy.py", line 194, in _run_module_as_main
>     return _run_code(code, main_globals, None,
>   File "/root/miniconda3/envs/webshop/lib/python3.8/runpy.py", line 87, in _run_code
>     exec(code, run_globals)
>   File "/root/workspace/src/server/task_worker.py", line 256, in <module>
>     asyncio_task = InstanceFactory.parse_obj(conf[args.name]).create()
>   File "/root/workspace/src/typings/general.py", line 34, in create
>     mod = __import__(path, fromlist=[self.module.split(".")[-1]])
> ModuleNotFoundError: No module named 'src.server.tasks.webshop_docker'
> ```
> 
> this seems to be caused by this `ln: failed to create symbolic link '/root/workspace/src/server/tasks/webshop_docker': File exists ` I checked the docker image it already has `/root/webshop` so the `ln -s` command won't work

Would you try removing `src/server/tasks/webshop_docker` and start it again?

**Comment by nlpcat** [comment]
@zhc7 can I ask whether this is already installed in the docker image? https://github.com/THUDM/AgentBench/tree/main/src/server/tasks/webshop#webpage-environment-html-mode

**Comment by nlpcat** [comment]
@zhc7 yes. it works after I removed src/server/tasks/webshop_docker file

**Comment by nlpcat** [comment]
@zhc7 there seems to be a bug if we assign webshop-dev but webshop-std can work.
```
raise AgentBenchException(result.text, result.status_code, self.name)
src.typings.exception.AgentBenchException: ('{"detail":"Error: Task does not exist"}', 400, 'webshop-dev')
```

**Comment by zhc7** [comment]
> @zhc7 yes. it works after I removed src/server/tasks/webshop_docker file

Thanks. I'll remove this file from the repo.

**Comment by zhc7** [comment]
> @zhc7 there seems to be a bug if we assign webshop-dev but webshop-std can work.
> 
> ```
> raise AgentBenchException(result.text, result.status_code, self.name)
> src.typings.exception.AgentBenchException: ('{"detail":"Error: Task does not exist"}', 400, 'webshop-dev')
> ```

Have you started `webshop-dev` task_worker? Note that `webshop-std` and `webshop-dev` are independent. Try executing `python -m src.start_task -s webshop-dev -p 7000`  to manually start a task worker or add `webshop-dev` to start_task.yaml.

**Comment by nlpcat** [comment]
yes. it works after changing the start_task.yaml. but ltp task failed with this error ` File "/root/workspace/src/client/agents/claude_agent.py", line 1, in <module>
    import anthropic
ModuleNotFoundError: No module named 'anthropic'`. the docker image might need to get updated?

**Comment by Longin-Yu** [comment]
We've just uncovered this issue, and it may take some time to pinpoint it due to the complexity of the `card_game` task.

**Comment by nlpcat** [comment]
@Longin-Yu may I ask when we can fix the card game test? thanks

**Comment by Longin-Yu** [comment]
We have located this bug and we are working on fixing it. We expect an update within this week.

**Comment by zhc7** [comment]
@nlpcat Hi, we've fixed the problem in [a069c7c](https://github.com/THUDM/AgentBench/commit/a069c7cb4ab81f49abf252eb78c88fd0bebb74af). Welcome to try!

**Comment by Longin-Yu** [comment]
Feel free to reopen this issue if it persists.
